src/main/java/edu/ucsb/cs156/example/services/UCSBCurriculumService.java

```diff
@@ -0,0 +1,105 @@
+package edu.ucsb.cs156.example.services;
+import java.util.Map;
+import java.util.Arrays;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.web.client.RestTemplateBuilder;

```

Comment:
nitpick: can we keep all the `org.springframework` imports together and alphabetized?